src/pkg/api/handlers.go

```diff
@@ -776,6 +776,21 @@ func getVirtualService(cache *resources.Cache) func(w http.ResponseWriter, r *ht
 	return rest.BindCustomResource(cache.VirtualServices, cache)
 }
 
+// @Description Get Gateways
+// @Tags gateways
+// @Accept  html
+// @Produce text/event-stream,json
+// @Success 200
+// @Router /api/v1/resources/networks/gateways/{id} [get]
```

Comment:
```suggestion
// @Router /api/v1/resources/networks/gateways [get]
```
tested and as is it fails in swagger because of the {uid}